wordle-web/src/router/index.ts

```diff
@@ -22,6 +23,16 @@ const router = createRouter({
       component: LeaderboardView
     }
     ,
+    {
+      path: '/lasttenwords',
+      name: 'lastTenWords',
+      component: LastTenWords
```

Comment:
These are typically hyphenated 
```suggestion
      path: '/last-ten-words',
      name: 'last-ten-words',
      component: LastTenWords
```